validate.C

```diff
@@ -291,63 +289,98 @@ void calomet(TString algo, TString var, bool doLog10 = false){
-  photon("showerShapeVariables().sigmaIphiIphi",cName, true);
-  photon("showerShapeVariables().e2nd",cName, true);
-  photon("showerShapeVariables().eTop",cName, true);
-  photon("showerShapeVariables().eLeft",cName, true);
-  photon("showerShapeVariables().eRight",cName, true);
-  photon("showerShapeVariables().eBottom",cName, true);
-  photon("showerShapeVariables().e1x3",cName, true);
-  photon("showerShapeVariables().e2x2",cName, true);
-  photon("showerShapeVariables().e2x5Max",cName, true);
-  photon("showerShapeVariables().e2x5Left",cName, true);
-  photon("showerShapeVariables().e2x5Right",cName, true);
-  photon("showerShapeVariables().e2x5Top",cName, true);
-  photon("showerShapeVariables().e2x5Bottom",cName, true);
+  photon("e1x5", cName,tName);
+  photon("e2x5", cName,tName);
+  photon("e3x3", cName,tName);
+  photon("e5x5", cName,tName);
+  photon("full5x5_e1x5", cName,tName);
+  photon("full5x5_e2x5Max", cName,tName);
+  photon("full5x5_e5x5", cName,tName);
+
+  photon("maxEnergyXtal", cName,tName);
+  photon("sigmaEtaEta", cName,tName);
+  photon("sigmaIetaIeta", cName,tName);
+  photon("r1x5", cName,tName);
+  photon("r2x5", cName,tName);
+  //  photon("r9", cName,tName);
+  photon("mipChi2", cName,tName);
+  photon("mipNhitCone", cName,tName);
+  photon("ecalRecHitSumEtConeDR03", cName,tName);
+  photon("hcalTowerSumEtConeDR03", cName,tName);
+  photon("hcalDepth1TowerSumEtConeDR03", cName,tName);
+  photon("trkSumPtSolidConeDR03", cName,tName);
+  photon("trkSumPtHollowConeDR03", cName,tName);
+  photon("chargedHadronIso", cName,tName);
+  photon("neutralHadronIso", cName,tName);
+  photon("photonIso", cName,tName);
+  photon("sumChargedParticlePt", cName,tName);
+  photon("sumNeutralHadronEtHighThreshold", cName,tName);
+  photon("sumPhotonEtHighThreshold", cName,tName);
+  photon("sumPUPt", cName,tName);
+  photon("nClusterOutsideMustache", cName,tName);
+  photon("etOutsideMustache", cName,tName);
+  photon("pfMVA", cName,tName);
+  photon("showerShapeVariables().effSigmaRR",cName,tName, true);
+  photon("showerShapeVariables().sigmaIetaIphi",cName,tName, true);
+  photon("showerShapeVariables().sigmaIphiIphi",cName,tName, true);
+  photon("showerShapeVariables().e2nd",cName,tName, true);
+  photon("showerShapeVariables().eTop",cName,tName, true);
+  photon("showerShapeVariables().eLeft",cName,tName, true);
+  photon("showerShapeVariables().eRight",cName,tName, true);
+  photon("showerShapeVariables().eBottom",cName,tName, true);
+  photon("showerShapeVariables().e1x3",cName,tName, true);
+  photon("showerShapeVariables().e2x2",cName,tName, true);
+  photon("showerShapeVariables().e2x5Max",cName,tName, true);
+  photon("showerShapeVariables().e2x5Left",cName,tName, true);
+  photon("showerShapeVariables().e2x5Right",cName,tName, true);
+  photon("showerShapeVariables().e2x5Top",cName,tName, true);
+  photon("showerShapeVariables().e2x5Bottom",cName,tName, true);
 
   //  photon("r9", cName);
   photon("mipChi2", cName);
```

Comment:
shouldn't this and lines below be removed? they have a match with cName,tName above